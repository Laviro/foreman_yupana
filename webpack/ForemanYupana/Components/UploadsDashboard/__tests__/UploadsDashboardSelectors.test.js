import { testSelectorsSnapshotWithFixtures } from 'react-redux-test-utils';
import {
  selectUploadsDashboard,
  selectBool,
} from '../UploadsDashboardSelectors';

const state = {
  ForemanYupana: {
    uploadsDashboard: {
      bool: false,
    },
  },
};

const fixtures = {
  'should return UploadsDashboard': () => selectUploadsDashboard(state),
  'should return UploadsDashboard bool': () => selectBool(state),
};

describe('UploadsDashboard selectors', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
