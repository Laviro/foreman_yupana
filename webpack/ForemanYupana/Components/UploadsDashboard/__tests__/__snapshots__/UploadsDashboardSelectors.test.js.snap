// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UploadsDashboard selectors should return UploadsDashboard 1`] = `
Object {
  "bool": false,
}
`;

exports[`UploadsDashboard selectors should return UploadsDashboard bool 1`] = `false`;
